src/AppInstallerRepositoryCore/Microsoft/Schema/Portable_1_0/PortableTable.h

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "SQLiteWrapper.h"
+#include "SQLiteStatementBuilder.h"
+#include <string_view>
+
+namespace AppInstaller::Repository::Microsoft::Schema::Portable_V1_0
+{
+    // File type enum of the portable file
+    enum class PortableFileType
+    {
+        Unknown,
+        File,
+        Directory,
+        Symlink
+    };
+
+    // Column type enum of the portable table
+    enum class PortableColumnType
+    {
+        FilePath,
+        FileType,
+        SHA256,
+        SymlinkTarget,
+        IsCreated
+    };
+
+    // Metadata representation of a portable file placed down during installation
+    struct PortableFile
+    {
+        std::string FilePath;

```

Comment:
Why not a path?  I think for other reasons you will want a canonicalized path, so I would not make this a public field and instead have a set function that enforces that.

Reply:
Changed to path and added a setting function to enforce the canonicalized path